monitoring/ceph-mixin/dashboards_out/radosgw-overview.json

```diff
@@ -785,7 +785,7 @@
             },
             {
                "expr": "sum(irate(haproxy_backend_response_errors_total{proxy=~\"backend\",instance=~\"$ingress_service\"}[5m])) by (instance)",
-               "format": "time_series",
+               "format": "",
```

Comment:
I still see this missing multiple times in this dashboard (see below).

Reply:
Updated